JULIERAJ/KIDS-FIRST-Front
# Bugfix/27/display username

PR Template Frontend
One Line Description
  - This pull request updates the DashboardHeader component to display the logged-in user's full name and initials.
  
Requirements
The code changes should:
Extract and display the user's first name from the provided user object.
Display the user's initials inside a styled circle container.
Ensure the DashboardHeader component is properly receiving and handling the user object.

Notes
  - This update is necessary to provide a personalized user experience by displaying the user's name and initials on the dashboard header.

The changes were made to the DashboardHeader component.

Test Steps
    - Ensure the backend server is running and serving user authentication details correctly.
    - Log in to the application to have a valid session and user data.
    - Navigate to the dashboard.
    - Check that the user's first name is displayed correctly in the header.
    - Verify that the initials of the user are displayed inside the circle container.

UI/UX Outcome
![image](https://github.com/user-attachments/assets/f067552a-a958-44fe-b2b1-0632ba2f6a85)


Checklist
